examples/sqlite/src/query_root.rs

```diff
@@ -14,229 +12,24 @@ pub fn schema(
-                crate::entities::store::Relation::Staff.def(),
-            ),
-    ]);
-    builder.register_entity::<crate::entities::payment::Entity>(vec![
-        entity_object_relation_builder
-            .get_relation::<crate::entities::payment::Entity, crate::entities::customer::Entity>(
-                "customer",
-                crate::entities::payment::Relation::Customer.def(),
-            ),
-        entity_object_relation_builder
-            .get_relation::<crate::entities::payment::Entity, crate::entities::rental::Entity>(
-                "rental",
-                crate::entities::payment::Relation::Rental.def(),
-            ),
-        entity_object_relation_builder
-            .get_relation::<crate::entities::payment::Entity, crate::entities::staff::Entity>(
-                "staff",
-                crate::entities::payment::Relation::Staff.def(),
-            ),
-    ]);
-    builder.register_entity::<crate::entities::address::Entity>(vec![
-        entity_object_relation_builder
-            .get_relation::<crate::entities::address::Entity, crate::entities::city::Entity>(
-                "city",
-                crate::entities::address::Relation::City.def(),
-            ),
-        entity_object_relation_builder
-            .get_relation::<crate::entities::address::Entity, crate::entities::customer::Entity>(
-                "customer",
-                crate::entities::address::Relation::Customer.def(),
-            ),
-        entity_object_relation_builder
-            .get_relation::<crate::entities::address::Entity, crate::entities::staff::Entity>(
-                "staff",
-                crate::entities::address::Relation::Staff.def(),
-            ),
-        entity_object_relation_builder
-            .get_relation::<crate::entities::address::Entity, crate::entities::store::Entity>(
-                "store",
-                crate::entities::address::Relation::Store.def(),
-            ),
-    ]);
+
+    seaography::register_entity!(builder, &CONTEXT, film_actor);
+    seaography::register_entity!(builder, &CONTEXT, rental);
+    seaography::register_entity!(builder, &CONTEXT, category);
+    seaography::register_entity!(builder, &CONTEXT, staff);
+    seaography::register_entity!(builder, &CONTEXT, country);
+    seaography::register_entity!(builder, &CONTEXT, film);
+    seaography::register_entity!(builder, &CONTEXT, actor);
+    seaography::register_entity!(builder, &CONTEXT, language);
+    seaography::register_entity!(builder, &CONTEXT, city);
+    seaography::register_entity!(builder, &CONTEXT, inventory);
+    seaography::register_entity!(builder, &CONTEXT, film_text);
+    seaography::register_entity!(builder, &CONTEXT, film_category);
+    seaography::register_entity!(builder, &CONTEXT, customer);
+    seaography::register_entity!(builder, &CONTEXT, store);
+    seaography::register_entity!(builder, &CONTEXT, payment);
+    seaography::register_entity!(builder, &CONTEXT, address);
```

Comment:
```suggestion
    seaography::register_entity!(
        builder,
        &CONTEXT,
        [
            film_actor,
            rental,
            category,
            staff,
            country,
            film,
            actor,
            language,
            city,
            inventory,
            film_text,
            film_category,
            customer,
            store,
            payment,
            address
        ]
    );
```

Reply:
we can have an additional macro called `register_entities`

Reply:
I have gone with the register_entities solution

Reply:
That's even cooler